### ISSUE TITLE
Corrupted symlink targets

### ISSUE THREAD

**Issue by WhiteWind** [opendedup/sdfs#9]
I've installed version 3.1.9 on debian wheezy. This involved rebuilding libfuse with threads patch and libjavafs.so because of old GLIBC.

sdfs launched successfully and I performed a backup of my database.
There were symlinked tables and all of them become corrupt. 
I mean, symlink targets were corrupt. They become 61 byte files with content:
`¬н..sr.'org.opendedup.sdfs.io.MetaDataDedupFileА-HAВsА5...xpx`

**Comment by opendedup** [comment]
Hi - can you give me a example of the symlinks? Like run ls -lah from the
sdfs directory?

**Comment by WhiteWind** [comment]
They are symlinks:
```
# ls -lah /opt/sdfs/volumes/dedup/files/2016-05-27-00/mysql/kombat/
lrwxrwxrwx  1 mysql mysql   40 May 27 00:32 paths_2016@002d05.MYI -> /mnt/ssd/kombat/kombat/paths_2016-05.MYI
```

**Comment by opendedup** [comment]
Mikhail - Can you tell me how you did the backup? I am trying to replicate
the issue here and need detail on how to reproduce the issue.

I did symlink testing this morning and was not able to reproduce the issue.

Sam

**Comment by WhiteWind** [comment]
I did cp -av /opt/mysqlvol/mysql /opt/backup/2016-05-07-00/
SDFS was mounted on /opt/backup
Note, that symlink targets are not inside /opt/mysqlvol

I'm using: 
patched fuse 2.9.0 (I deleted /usr/share/sdfs/bin/libfuse.so)
sdfs 3.1.9 from http://www.opendedup.org/downloads/sdfs-latest.deb
libc6 2.13-38+deb7u6

I had to recompile libjavafs.so from source, I used git tag 3.1.7, because tag 3.1.9 does not exists.
Should I recompile java code as well?

**Comment by WhiteWind** [comment]
Any news?
